.github/workflows/release.yml

```diff
@@ -32,10 +32,10 @@ jobs:
       run: python -m build
 
     - name: publish
-      uses: pypa/gh-action-pypi-publish@release/v1
+      uses: pypa/gh-action-pypi-publish@v1.8.10/v1
```

Comment:
I _think_ we want to keep this as `release/v1`, rather than using a specific tag.

Reply:
Ah yes, this was a mistake. I have a script to update tags, but it doesn't handle this one very well. I'll revert it.